src/Series.cpp

```diff
@@ -1277,7 +1277,13 @@ void Series::readOneIterationFileBased(std::string const &filePath)
     if (version == "1.0.0" || version == "1.0.1" || version == "1.1.0")
         pOpen.path = auxiliary::replace_first(basePath(), "/%T/", "");
     else
-        throw std::runtime_error("Unknown openPMD version - " + version);
+        throw error::ReadError(
+            error::AffectedObject::File,
+            error::Reason::UnexpectedContent,
+            std::nullopt,
+            "Unknown openPMD version - " + version +
+                ". Consider upgrading your installation of the openPMD-api "
+                "(https://openpmd-api.readthedocs.io).");
```

Comment:
Is there a way to link https://openpmd-api.readthedocs.io/en/0.15.2/install/install.html here without hardcoding the version or using latest? (latest is the dev)

Reply:
I guess this idea does not make sense as the docs structure could change between releases.
Since we are trying to link to a newer version than $current for the purpose of upgrading, the root link should be fine.